docs/2.10.0/docs/canton/usermanual/security.rst

```diff
@@ -320,6 +320,29 @@ A fingerprint of a key can be retrieved from the list of public keys:
    :end-before: user-manual-entry-end: ListPublicKeys
    :dedent:
 
+.. _deleting-canton-keys:
+
+Deleting Canton Node Keys
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Deletion of keys, in particular, private keys is not done automatically in Canton. When a key is rotated, the previous
+key remains stored, but is disabled and no longer used. To permanently delete a key, use the following Canton console
+command:
+
+.. literalinclude:: /canton/includes/mirrored/enterprise/app/src/test/scala/com/digitalasset/canton/integration/tests/security/topology/KeyManagementIntegrationTest.scala
+   :language: scala
+   :start-after: user-manual-entry-begin: DeletePrivateKey
+   :end-before: user-manual-entry-end: DeletePrivateKey
```

Comment:
Canton's main does not seem to have those delimiters yet. Can you point me to a PR that adds them?

Reply:
https://github.com/DACH-NY/canton/pull/22374 and https://github.com/DACH-NY/canton/pull/22375